app/views/providers/allocations/requests/_yes_requested.html.erb

```diff
@@ -5,28 +5,30 @@
     <div class="govuk-panel govuk-panel--confirmation govuk-!-margin-bottom-7" data-qa="page-heading">
       <h1 class="govuk-panel__title">Request sent</h1>
       <p class="govuk-panel__body govuk-!-margin-bottom-0">
-        You’ve confirmed that <%= @training_provider.provider_name %> would like to offer PE in 2021/22
+        You’ve confirmed that <%= @training_provider.provider_name %> would like to offer PE in 2021 - 2022.
       </p>
     </div>
     <h2 class="govuk-heading-m">What happens now?</h2>
     <p class="govuk-body">
-      You have until 10 July to change this request. On or after 10 July,
-      we’ll send an email to your organisation confirming the number of places.
+      You have until 10 July to change this request. After this date,  we’ll send an email to your
```

Comment:
```suggestion
      You have until 10 July 2020 to change this request. After this date,  we’ll send an email to your
```

missing 2020?

Reply:
This is the content that I was giving to add to it. I suppose it should be obvious because they wouldn't be accessing this page in 2021 beyond